isis/src/control/objs/BundleUtilities/BundleObservationSolveSettings.cpp

```diff
@@ -178,24 +174,54 @@ namespace Isis {
      }
     }
 
-    if (csolve != "NONE") {
-     if (scParameterGroup.hasKeyword("CAMERA_ANGLES_SIGMA"))
-       m_anglesAprioriSigma.append((double)(scParameterGroup.findKeyword("CAMERA_ANGLES_SIGMA")));
-     if (scParameterGroup.hasKeyword("CAMERA_ANGULAR_VELOCITY_SIGMA"))
-       m_anglesAprioriSigma.append(
+    if (m_instrumentPointingSolveOption) {
```

Comment:
This is an enumeration I would check `m_instrumentPointingSolveOption != NoPointingFactors`